src/main/java/bio/terra/cli/app/DockerCommandRunner.java

```diff
@@ -92,20 +86,10 @@ protected int runToolCommandImpl(String command, Map<String, String> envVars)
       // if the ADC are set by a file, then make sure that file is mounted to the container and the
       // env var points to it if needed
       Optional<Path> adcCredentialsFile = AppDefaultCredentialUtils.getADCBackingFile();
-      if (adcCredentialsFile.isPresent()) {
-        if (adcCredentialsFile.get().equals(AppDefaultCredentialUtils.getDefaultGcloudADCFile())) {
-          logger.info(
-              "ADC backing file is in the default location and is already mounted in the gcloud config directory");
-        } else {
-          // mount the pet SA key file to the container in the .terra directory
-          // e.g. ADC file (host) $HOME/pet-sa-key.json -> (container)
-          // CONTAINER_HOME_DIR/.terra/pet-keys/[user id]/application_default_credentials.json
-          Path adcFileOnContainer = getADCFileOnContainer();
-          bindMounts.put(adcFileOnContainer, adcCredentialsFile.get());
-
-          // set the env var to point to the file, since it's not in the default gcloud location
-          envVars.put("GOOGLE_APPLICATION_CREDENTIALS", adcFileOnContainer.toString());
-        }
+      if (adcCredentialsFile.isPresent()
+          && adcCredentialsFile.get().equals(AppDefaultCredentialUtils.getDefaultGcloudADCFile())) {
+        logger.info(
+            "ADC backing file is in the default location and is already mounted in the gcloud config directory");
       } else {
         logger.info("ADC set by metadata server.");
```

Comment:
Maybe indicate that this is the VM's local metadata server, or however you say it.